tests/MartinGeorgiev/Doctrine/DBAL/Types/TextArrayTest.php

```diff
@@ -46,11 +46,16 @@ public function validTransformations(): array
             ],
             [
                 'phpValue' => [
+                    1,
+                    '2',
+                    'text',
                     'some text here',
                     'and some here',
-                    'even here there is text',
+                    "single-quotes ain't no worry, right Alexander O'Vechkin?",
```

Comment:
Can you please add examples with double- and triple-repeated quotes (both with `'` and `"`) so the behaviour can be confirmed as fool-proof?